docs/UserGuide.md

```diff
@@ -206,7 +224,13 @@ Examples:
     * will match a student named `benny tan`, with address `West Jurong`, and email `benny.tan@gmail.com`
     * will match a student name: `benjamin`, with address `yishun east ave 1`, and email: `benj@gmail.com`
 
-### Filtering students by tags: `filter`
+#### Viewing all tags: `tag`
+
+Shows all the tags that user has created in TAB.
+
+Format: `tag`
+
+#### Filtering students by tags: `filter`
```

Comment:
same with `find` and `filter` since they are some sort of search and view function , so perhaps could have a subsection for `search` under `student` but this can be done at a later time so

Reply:
ya I think can think again later

Reply:
I added a `Finding Students` section